config/locales/en.yml

```diff
@@ -85,6 +85,8 @@ en:
     pull_requests: Wonder what people are up to this holiday? Check out these pull requests,
       or submit a pull request now and find yours here!
     synchronize: Synchronize Pull Requests
+    tweet_progress_singular: I’ve sent 1 pull request in December! Join me in giving back to open source with @24pullrequests.
+    tweet_progress_plural: I’ve sent %{count} pull requests in December! Join me in giving back to open source with @24pullrequests.
```

Comment:
You gotta add those keys to all other locales (empty ones should fallback to :en locale)

Reply:
Also maybe it's not a bad time to start structuring locale files properly. See "Lazy lookup": http://guides.rubyonrails.org/i18n.html#looking-up-translations

So the change is `t(".tweet_progress")` and locale file will have something like this:

```yml
en:
  dashboards:
    show:
      tweet_progress: "I've sent pull requests!"
```

Reply:
I think the changing of structure is a little out of scope, as it would make this instance inconsistent with the rest of the codebase. You should make an issue for that though so someone could restructure in one swoop.

Reply:
There is an issue: https://github.com/24pullrequests/24pullrequests/issues/2125

There's no structure to begin with, so adding new translations properly will mean less work to clean up later.